### PR TITLE
Issue #11214: Specified violation messages in EmptyLineSeparatorCheck

### PR DESCRIPTION
**Pull request by coderdeadpool**
Issue #11214

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorWithComments.java [hunk 1]
```diff
@@ -69,42 +69,42 @@
...
 
-// violation
+// violation ''//' has more than 1 empty lines before.'
 import java.lang.StringBuilder;
 
 
-/*
- * // violation above
+/* // violation ''/*' has more than 1 empty lines before.'
+ *
```
Comment (Vyom-Yadav) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorWithComments.java:87]: The violation message is interpreted as a regular expression, you need to escape `*` as it is a special character. You may escape using `\` or may use https://stackoverflow.com/questions/30776860/regular-expression-q-e

**Comment by coderdeadpool** [comment]
I am closing this PR and creating a new one, due to some error in merging the conflicts :( .

**Comment by nrmancuso** [comment]
@coderdeadpool please do not close/ open new PRs. This generates a bunch of noise and all context and review from previous PR is lost.

**Comment by coderdeadpool** [comment]
@nrmancuso Sorry for that. But i tried to fix the conflicts and end up creating more mess.
I will keep that in mind and avoid doing this.

**Comment by nrmancuso** [comment]
No need to keep in mind, you can reopen PR and work through git issues

**Comment by coderdeadpool** [comment]
@nrmancuso I had to create a new pull request because I deleted the previous repository.

**Pull request closed without merging**